src/networks.json

```diff
@@ -1539,6 +1539,20 @@
     "start": 7282345,
     "logo": "ipfs://QmRHB9TqMdVHY392vYiv8sTJ7VHShkq5FT6nS9fPuUNBf1"
   },
+  "7363": {
+    "key": "7363",
+    "name": "DynoChain",
+    "shortName": "DND",
+    "chainId": 7363,
+    "network": "mainnet",
+    "multicall": "0x70F9C674677Cb915042C11fcd4D3418A99aa7A3D",
+    "rpc": [
+      "https://rpc.dynochain.io"
+    ],
+    "explorer": "https://dynoscan.io",
```

Comment:
plz verify and publish the code to explorer

Reply:
We are updating explorer for some reason contract wont verify. Working on it now.

Reply:
contract verified  https://dynoscan.io/address/0x70F9C674677Cb915042C11fcd4D3418A99aa7A3D/contracts#address-tabs